keyboards/hubble/info.json

```diff
@@ -0,0 +1,90 @@
+{
+    "manufacturer": "Anton Chernenko",
+    "keyboard_name": "hubble",
+    "maintainer": "ch3rny",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F5", "F6", "B4", "E6", "D7", "C6", "D4", "D0"],
+        "rows": ["D1", "F4", "F7", "B5", "B1", "B3", "B6", "B2"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "vid": "0x1234",
```

Comment:
This vendor ID is in use, and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x1234&p=&n=

The further pass 0x4000, the better.

Reply:
updated